database/migrations/2_create_blocks_table.up.cql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS blocks (
+    block_hash TEXT PRIMARY KEY,
+    raw_block BLOB
+);
```

Comment:
We would probably like to keep some block metadata separately for easier access, but at the moment I don't yet know what metadata is required by the verification tool. Let's stay flexible with this table, allowing for easy addition of columns in the future, ok?

Reply:
There is a database migration mechanism in place, so adding new columns should not be a problem.